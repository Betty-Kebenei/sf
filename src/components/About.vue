<template>
  <div class="about">
    <p>this is a about page</p>
  </div>
</template>
