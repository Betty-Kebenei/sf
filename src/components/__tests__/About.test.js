import { shallowMount } from '@vue/test-utils';
import About from '../About.vue';

describe('Navbar', () => {
  const wrapper = shallowMount(About);
  test('is a Vue instance', () => {
    expect(wrapper.isVueInstance()).toBeTruthy();
  });

  test('renders correctly', () => {
    expect(wrapper.element).toMatchSnapshot();
  });
});
