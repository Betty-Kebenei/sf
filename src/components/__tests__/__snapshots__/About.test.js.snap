// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar renders correctly 1`] = `
<div
  class="about"
>
  <h1>
    About Senders Fellowship
  </h1>
   
  <h2>
    Mission
  </h2>
   
  <p>
    
      To establish a stable African Sending structure.
  
  </p>
   
  <h2>
    Vision
  </h2>
   
  <p>
    
      To raise resources for kingdom advancement and provide
      moral and prayer support for those who go.
  
  </p>
   
  <h2>
    Values
  </h2>
   
  <ul>
    <li>
      Faith in Jesus Christ
    </li>
     
    <li>
      Commitment
    </li>
  </ul>
   
  <h2>
    Objectives
  </h2>
   
  <ul>
    <li>
      Be accountable to other people called to the marketplace.
    </li>
     
    <li>
      Raise resources for kingdom advancement.
    </li>
     
    <li>
      Provide moral support for those who go.
    </li>
     
    <li>
      Encourage one another.
    </li>
     
    <li>
      Brainstorm ideas.
    </li>
  </ul>
</div>
`;
